tests/plugins/test_check_user_settings.py

```diff
@@ -0,0 +1,84 @@
+"""
+Copyright (c) 2020 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+from __future__ import absolute_import
+
+from flexmock import flexmock
+import pytest
+
+from atomic_reactor.constants import PLUGIN_CHECK_USER_SETTINGS
+from atomic_reactor.inner import DockerBuildWorkflow
+from atomic_reactor.plugin import PreBuildPluginsRunner, PluginFailedException
+from atomic_reactor.plugins.pre_check_user_settings import CheckUserSettingsPlugin
+
+from tests.constants import TEST_IMAGE
+from tests.stubs import StubInsideBuilder, StubSource
+
+
+def mock_dockerfile(tmpdir, labels):
+    base = 'From fedora:30'
+    cmd = 'CMD /bin/cowsay moo'
+    extra_labels = [
+        'LABEL {}'.format(label)
+        for label in labels
+
+    ]
+    data = '\n'.join([base] + extra_labels + [cmd])
+    tmpdir.join('Dockerfile').write(data)
+
+
+def mock_workflow(tmpdir, for_orchestrator=False):
+    workflow = DockerBuildWorkflow(
+        TEST_IMAGE,
+        source={"provider": "git", "uri": "asd"}
+    )
+    workflow.source = StubSource()
+    builder = StubInsideBuilder().for_workflow(workflow)
+    builder.set_df_path(str(tmpdir))
+    builder.tasker = flexmock()
+    workflow.builder = flexmock(builder)
+
+    if for_orchestrator:
+        workflow.buildstep_plugins_conf = [{'name': PLUGIN_CHECK_USER_SETTINGS}]
```

Comment:
I might be missing something, but shouldn't this be `PLUGIN_BUILD_ORCHESTRATE_KEY`?

Reply:
Hmm, you are right, I;m thinking if this section is even needed in test

Reply:
removed